website/cue/reference/components/sinks/base/pulsar.cue

```diff
@@ -159,4 +159,9 @@ base: components: sinks: pulsar: configuration: {
 		required:    true
 		type: string: syntax: "literal"
 	}
+	producer_name: {
+		description: "The name of the producer. If not specified, the default name will be used."
```

Comment:
It would be good to make explicit what the default name is.

Reply:
Hah, you have the same questions as me :) Unfortunately, I did not find in the official documentation, what is the default name is used. The only what I've found is https://github.com/apache/pulsar/issues/14272#issuecomment-1039017004

And simply says that name starts with "producer" and nothing more. I even have tried to dig into the code but without success.

Since the comment is not an official documentation and the default name could be changed in the future without any commit for stability from the Apache Pulsar devs, I did not add the details about the default name.

Reply:
Fair enough. Then I would suggest something like "the default name assigned by Pulsar" to make explicit that it isn't a _Vector_ default.

Reply:
Fixed.